vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -92,27 +91,22 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
                 it.bind(holder.messageView)
             }
         }
-        val textFuture = if (bindingOptions?.canUseTextFuture.orFalse()) {
-            PrecomputedTextCompat.getTextFuture(
-                    message?.charSequence ?: "",
-                    TextViewCompat.getTextMetricsParams(holder.messageView),
-                    null)
-        } else {
-            null
-        }
-        markwonPlugins?.forEach { plugin -> plugin.beforeSetText(holder.messageView, message?.charSequence as Spanned) }
```

Comment:
This line (`markwonPlugins...`) has been removed by mistake during the merge I guess

Reply:
ah :facepalm: I got confused by https://github.com/vector-im/element-android/pull/4790/files#diff-473563ebe7dac7363901c094cc094bb438921f76a16bf13a020eefbff15d0c24R100 didn't realise there were two

Reply:
There are 2 different method `beforeSetText` and a `afterSetText`, I have no idea what they are doing TBH

Reply:
https://github.com/vector-im/element-android/pull/4790/commits/22bab47dad9072787326ae11a8587138df134c5f